Cargo.toml

```diff
@@ -20,6 +20,9 @@ members = [
     "wgpu-macros",
     "wgpu-types",
     "wgpu",
+    "wgpu-core-target-feature-conditionals/arch-wasm32",
+    "wgpu-core-target-feature-conditionals/native-non-apple",
+    "wgpu-core-target-feature-conditionals/vendor-apple",
```

Comment:
Could we consider shorter names for these, perhaps with slight reordering?

```suggestion
    "wgpu-core-target-features/arch-wasm32",
    "wgpu-core-target-features/native-non-apple",
    "wgpu-core-target-features/vendor-apple",
    "wgpu",
```

Reply:
yeah those names are definitely too crazy right now

Reply:
I also have thoughts on the naming scheme that I'll fully think through once this is out of draft